install.sh

```diff
@@ -11,9 +11,10 @@ source "$(dirname $0)/install/_lib.sh"  # does a `cd .../install/`, among other
 
 # Pre-flight. No impact yet.
 source parse-cli.sh
-source detect-platform.sh
 source dc-detect-version.sh
 source error-handling.sh
+# We set the trap at the top level so that we get better tracebacks.
+trap_with_arg cleanup ERR INT TERM EXIT
```

Comment:
What if there's an error in parse cli or dc-detect?

Reply:
Our error handling depends on Docker specifics, so dc detect should come before.

Parse CLI is pretty thin, fine to leave it where it is.